<html><head>
<style>
    .highlight { color: blue; font-weight: bold; }
    body { font-family: Arial, sans-serif; }
    h1 { text-align: center; }
    h2 { color: darkred; }
    .sentence { margin-bottom: 10px; }
</style>
</head><body>
<h1>Highlighted GST ML Predictions for Full-Length Articles</h1><h2>PMC10371642_TITLE_Paragraph1</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> Retracted: RUNX3-Regulated GALNT6 Promotes the Migration and Invasion of Hepatocellular Carcinoma Cells by Mediating O-Glycosylation of MUC1.</p><h2>PMC2391110_ABSTRACT</h2><p class="sentence"><b>Sentence 0:</b> Overexpression and altered glycosylation of MUC1 in malignant mesothelioma.</p><p class="sentence"><b>Sentence 1:</b> Current interest in the MUC1/EMA mucin relates to its role in malignancy, and its potential as a therapeutic target.</p><p class="sentence"><b>Sentence 2:</b> MUC1/EMA expression has been observed in the majority of epithelioid mesotheliomas.</p><p class="sentence"><b>Sentence 3:</b> However, little is known of the characteristics of MUC1/EMA in mesothelioma.</p><p class="sentence"><b>Sentence 4:</b> Herein, we studied the cell surface and soluble expression of the MUC1/EMA glycoprotein, and determined the mRNA and genomic expression profiles in mesothelioma.</p><p class="sentence"><b>Sentence 5:</b> We found that the anti-MUC1 antibody, E29, was the most diagnostically useful of seven antibody clones examined with a sensitivity of 84% (16 out of 19 cases) and no false positive results.</p><p class="sentence"><b>Sentence 6:</b> MUC1 mRNA expression was significantly higher in mesothelioma samples than in benign mesothelial cells.</p><p class="sentence"><b>Sentence 7:</b> No amplification of the MUC1 gene was observed by FISH.</p><p class="sentence"><b>Sentence 8:</b> Seven of 9 mesothelioma samples expressed MUC1-secreted mRNA isoform in addition to the archetypal MUC1/transmembrane form.</p><p class="sentence"><b>Sentence 9:</b> CA15.3 (soluble MUC1) levels were significantly higher in the serum of mesothelioma patients than in healthy controls but were not significantly different to levels in patients with benign asbestos-related disease.</p><p class="sentence"><b>Sentence 10:</b> CA15-3 in effusions could differentiate malignant from benign effusions but were not specific for mesothelioma.</p><p class="sentence"><b>Sentence 11:</b> Thus, as in other cancers, alterations in MUC1 biology occur in mesothelioma and these results suggest that specific MUC1 characteristics may be useful for mesothelioma diagnosis and should also be investigated as a potential therapeutic target.</p><h2>PMC2391110_DISCUSS_Paragraph1</h2><p class="sentence"><b>Sentence 0:</b> DISCUSSION.</p><p class="sentence"><b>Sentence 1:</b> Normal/reactive nonmalignant mesothelial cells express predominantly the full-length <span class="highlight">tandem-repeat</span> containing MUC1-TM, which can be detected on the cell membrane by the Mc5 antibody.</p><p class="sentence"><b>Sentence 2:</b> In cases of malignant mesothelioma there is an increase in the total quantity of MUC1-TM mRNA expressed, a change in the type of MUC1 isoform produced, an alteration in the <span class="highlight">epitopes</span> of MUC1 expressed on cell surface and an increase in MUC1 gene product detectable in the circulation.</p><h2>PMC2391110_DISCUSS_Paragraph2</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> The Mc5 antibody was generated against delipidated human milk fat globule and recognises the DTRPAP <span class="highlight">epitope</span> in the VNTR of the MUC1-TM protein.</p><p class="sentence"><b>Sentence 2:</b> In the current study the Mc5 antibody recognised membrane associated antigen on mesothelial cells in over half of the effusion samples of nonmalignant origin examined and mesothelioma cells in all of the effusions associated with malignant mesothelioma.) demonstrated that the choice of antibody clone could greatly influence the accuracy of the use of anti-MUC1/EMA antibodies in distinguishing benign from malignant mesothelial cells in a diagnostic setting.</p><p class="sentence"><b>Sentence 3:</b> This may be one factor in the debate regarding the role of EMA immunohistochemistry in mesothelioma diagnosis.</p><p class="sentence"><b>Sentence 4:</b> The E29 clone and four of the other clones examined did not recognise MUC1 expressed on benign mesothelial cells.</p><p class="sentence"><b>Sentence 5:</b> The E29 clone was also generated against delipidated human milk fat globule and recognises an overlapping <span class="highlight">epitope</span> (the APDTRP <span class="highlight"><span class="highlight">epitope</span>)</span> to that recognised by Mc5.</p><p class="sentence"><b>Sentence 6:</b> It is noteworthy that the E29 clone stains normal breast, intestine and colon.</p><p class="sentence"><b>Sentence 7:</b> Alteration in MUC1 glycosylation has been reported in many malignancies.</p><p class="sentence"><b>Sentence 8:</b> The current antibody studies suggest that glycosylation of MUC1-TM is altered in malignant mesothelioma cells.</p><p class="sentence"><b>Sentence 9:</b> However, the actual nature of the alteration is unclear at present.</p><p class="sentence"><b>Sentence 10:</b> The finding of different staining profiles with the two antibodies, Mc5 and E29, which belong to the same group, recognise the same <span class="highlight">epitope</span> and are both affected by in vitro glycosylation needs to be further investigated.</p><h2>PMC2391110_DISCUSS_Paragraph3</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> Levels of MUC1 gene product in serum and effusions can be determined by several tests, the most common being the CA15-3, mucin-like associated antigen, CA27.29 and CA549 assays.</p><p class="sentence"><b>Sentence 2:</b> Differences between these tests derived from the monoclonal antibodies used to detect MUC1 <span class="highlight">epitopes</span> and the sensitivity of the antibodies to the level of glycosylation of the protein.</p><p class="sentence"><b>Sentence 3:</b> In the current study, the CA15-3 assay was used.</p><p class="sentence"><b>Sentence 4:</b> The major clinical role of CA15-3 biomarker is in monitoring breast carcinoma metastases and the evaluation of response to treatment.</p><p class="sentence"><b>Sentence 5:</b> <span class="highlight">CA15-3</span> has previously been found in several small scale studies to be elevated in the serum and in effusions of patients with mesothelioma.</p><p class="sentence"><b>Sentence 6:</b> In the current study, CA15-3 levels were significantly higher in the serum of mesothelioma patients than healthy controls; however, the finding that levels were elevated in patients with benign lung and pleural disease suggests that serum CA15-3 will not be useful as a diagnostic aid for mesothelioma.</p><p class="sentence"><b>Sentence 7:</b> Elevated levels of CA15-3 in effusions may be a strong indicator of malignancy in general as previously suggested.</p><h2>PMC2391110_DISCUSS_Paragraph4</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> The MUC1 gene product detected in the circulation by the CA15-3 assay contains the VNTR region, therefore the protein may be the archetypal MUC1-TM protein that has been released or cleaved from the cell surface, or may be one of the secreted isoforms.</p><p class="sentence"><b>Sentence 2:</b> There is some evidence to suggest the latter as the MUC1-SEC was detected by PCR in some of the samples derived from mesothelioma patients, but not from normal mesothelial samples or from cells in the nonmalignant pleural effusions examined.</p><p class="sentence"><b>Sentence 3:</b> Interestingly, the lack of MUC-SEC expression has been associated with ovarian cancer.</p><p class="sentence"><b>Sentence 4:</b> Our study used both clinical samples and, in an attempt to limit the influence of nonmesothelial cellular infiltrates, also in vitro cultured homogeneous pericardial cells.</p><p class="sentence"><b>Sentence 5:</b> It was encouraging to find generally a good correlation between the two sample types, as one concern was that alternative splicing of MUC1 might reflect changes induced by culture.</p><h2>PMC2391110_DISCUSS_Paragraph5</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> While the secreted splice form was detected in malignant mesothelial cells, it is of particular note that the expression of the full length, VNTR containing MUC1 gene product was 32-fold higher in mesothelioma than in normal cell preparations.</p><p class="sentence"><b>Sentence 2:</b> Evidence suggests that this overexpression was not due to a gene amplification event, but to an as yet unknown mechanism.</p><p class="sentence"><b>Sentence 3:</b> MUC1 overexpression is associated with poor prognosis in breast and lung adenocarcinoma; however, its influence in the prognosis of mesothelioma has not been reported.</p><h2>PMC2391110_DISCUSS_Paragraph6</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> Various forms of MUC1 are expressed in mesothelioma.</p><p class="sentence"><b>Sentence 2:</b> We suggest that the specific <span class="highlight">hypoglycosylated</span> form of the full length <span class="highlight">tandem</span> <span class="highlight">repeat</span> containing MUC1 protein product is a useful target for mesothelioma diagnosis, but possibly the expression of novel MUC1 <span class="highlight">epitopes</span> may represent a potential therapeutic target.</p><p class="sentence"><b>Sentence 3:</b> Currently there are promising MUC1 directed therapies being investigated in breast, ovarian and non small cell lung cancer.</p><p class="sentence"><b>Sentence 4:</b> These include therapies targeting MUC1-associated carbohydrates such as <span class="highlight">sialyl-Tn</span> (STn, Theratope), MUC1-pulsed dendritic cell vaccines and also the MUC1 peptide-based vaccine BLP-25.</p><p class="sentence"><b>Sentence 5:</b> Our findings suggest that the role of MUC1-directed therapy in mesothelioma should be investigated.</p><h2>PMC2391110_METHODS_Paragraph1</h2><p class="sentence"><b>Sentence 0:</b> MATERIALS AND METHODS Patients, samples and controls.</p><p class="sentence"><b>Sentence 1:</b> Serum, pleural effusion and surgically-excised tumour tissue samples were collected from patients following written informed consent.</p><p class="sentence"><b>Sentence 2:</b> All biospecimens were provided by the Australian Mesothelioma Tissue Bank, a member of the ABN-oncology group, which is supported by the National Health and Medical Research Council, Australia.</p><p class="sentence"><b>Sentence 3:</b> This study was approved by the human research ethics committees of Sir Charles Gairdner and Hollywood Hospitals, Perth, Western Australia.</p><p class="sentence"><b>Sentence 4:</b> The final diagnosis in all patients was confirmed by pathologists experienced in the diagnosis of mesothelioma and included clinical follow-up of all cases until death or to last citation in the Public Hospital database system (iSoft Clinical Manager) to confirm that the clinical pattern matched the diagnosis.</p><p class="sentence"><b>Sentence 5:</b> Mesotheliomas were classified as epithelial, sarcomatoid, mixed or, in cases where diagnosis was made on the basis of immunocytology and there was no histological follow-up, as unspecified.</p><h2>PMC2391110_METHODS_Paragraph2</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> Normal mesothelial cells were recovered from pericardial fluid obtained from patients undergoing intrathoracic surgery as described previously.</p><p class="sentence"><b>Sentence 2:</b> Serum samples were collected from healthy volunteers, and from patients with the asbestos-related lung disease, asbestosis, and the asbestos-related pleural disease, pleural plaques.</p><p class="sentence"><b>Sentence 3:</b> Pleural effusion samples were collected from patients with effusions caused by nonmesothelioma malignancy.</p><p class="sentence"><b>Sentence 4:</b> Effusions were classified as benign (i.e., nonmalignant) on the basis of cytological and immunohistochemical features, and further classified as exudates or transudates on the basis of Light's criteria.</p><p class="sentence"><b>Sentence 5:</b> Effusions were classified as being associated with an infection if micro-organisms were detected in the fluid.</p><p class="sentence"><b>Sentence 6:</b> Patient characteristics are listed in Table 1.</p><h2>PMC2391110_METHODS_Paragraph3</h2><p class="sentence"><b>Sentence 0:</b> Immunohistochemistry.</p><p class="sentence"><b>Sentence 1:</b> A tissue microarray was constructed by the Western Australian Research Tissue Network (Perth, Western Australia) from archival paraffin blocks.</p><p class="sentence"><b>Sentence 2:</b> The tissue microarray contained effusion cell pellets from 20 cases of malignant mesothelioma and 16 cases of benign-reactive pleural effusions.</p><p class="sentence"><b>Sentence 3:</b> Immunohistochemistry was performed using standard techniques.</p><p class="sentence"><b>Sentence 4:</b> Briefly, sections were deparaffinised with xylene and rehydrated in a graded series of ethanol.</p><p class="sentence"><b>Sentence 5:</b> Antigen-retrieval was performed for 10 min at high temperature in citrate buffer.</p><p class="sentence"><b>Sentence 6:</b> Sections were incubated with anti-EMA clones for 60 min and washed in PBS.</p><p class="sentence"><b>Sentence 7:</b> Immunodetection was performed using Envision+ Dual <span class="highlight">link</span> detection system (Dako, Glostrup, Denmark).</p><p class="sentence"><b>Sentence 8:</b> For negative controls, the primary antibody was omitted.</p><p class="sentence"><b>Sentence 9:</b> Antibodies were purchased from various suppliers (Table 2).</p><p class="sentence"><b>Sentence 10:</b> Some of these antibodies had been previously characterised by workshops organised by the International Society of Oncodevelopmental Biology and Medicine (ISOBM) in terms of the <span class="highlight">epitope</span> recognised within the VNTR and whether that recognition was dependent upon the glycosylation status of the <span class="highlight"><span class="highlight">epitope</span>.</span></p><p class="sentence"><b>Sentence 11:</b> These antibodies were assigned to various classes or clusters by the ISOBM workshops based upon their staining profile on normal breast tissue samples (summarised in Table 2).</p><h2>PMC2391110_METHODS_Paragraph4</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> Staining was assessed by three observers independently (JC, AS and GS).</p><p class="sentence"><b>Sentence 2:</b> A positive result was defined as the presence of membranous staining on tumour cells.</p><p class="sentence"><b>Sentence 3:</b> Staining intensity was graded semi-quantitatively as negative, equivocal (+/-), weak (1+), moderate (2+) or strong (3+).</p><p class="sentence"><b>Sentence 4:</b> Moderate and strong positivity was only assigned where the majority of cells showed positive staining.</p><p class="sentence"><b>Sentence 5:</b> Sensitivity was calculated as the total number of moderately and strongly stained mesothelioma samples divided by the total number of mesothelioma samples.</p><p class="sentence"><b>Sentence 6:</b> Specificity was calculated as the number of negative benign control samples divided by the total number of benign control samples.</p><p class="sentence"><b>Sentence 7:</b> False positive rate was calculated as the number of moderately and strongly stained benign samples divided by the total number of benign samples.</p><h2>PMC2391110_METHODS_Paragraph5</h2><p class="sentence"><b>Sentence 0:</b> Quantitative PCR.</p><p class="sentence"><b>Sentence 1:</b> RNA was extracted using Rneasy kits (Qiagen, Clifton Hill, Victoria, Australia), following the manufacturer's protocol. cDNA was generated in a standard reverse transcriptase reaction using oligo <span class="highlight">dT</span> to prime Superscript II (Invitrogen, Mt Waverly, Victoria, Australia).</p><p class="sentence"><b>Sentence 2:</b> Quantitative PCR was performed with specific primer sets (MUC1 forward 5'-AGACGTCAGCGTGAGTGATG-3'; reverse 5'-GACAGCCAAGGCAATGAGAT-3') on the iCycler iQ Real Time Detection System (BioRad, Gladesville, New South Wales, Australia) using the QuantiTech SYBR Green PCR Kit (Qiagen).</p><p class="sentence"><b>Sentence 3:</b> The primers correspond to nucleotides 4757-5017 of the human polymorphic epithelial mucin gene (GeneBank Accession number M61170) and amplify the MUC1-TM product.</p><p class="sentence"><b>Sentence 4:</b> The relative expression of MUC1-TM was calculated using the standard delta Ct formula as follows:.</p><h2>PMC2391110_METHODS_Paragraph6</h2><p class="sentence"><b>Sentence 0:</b> Fish. 4 mum sections from the tissue microarray were deparaffinised, dehydrated, microwave treated in citrate buffer (pH 6.0) for 10 min, digested in pepsin solution (4 mg ml-1 in 0.01N HCl) for 7 min at 37 C, rinsed in 0.3 M sodium chloride, 30 mM sodium citrate (pH 7.0) at room temperature for 5 min.</p><p class="sentence"><b>Sentence 1:</b> Dual-probe hybridisation was performed using three SpectrumGreen-labeled BAC clones which encompass the MUC1 gene at 1q22, plus a SpectrumOrange-labeled chromosome 1 centromeric probe (Vysis, Downers Grove, IL, USA) as a control.</p><p class="sentence"><b>Sentence 2:</b> Seventeen mesothelioma samples were available for FISH analysis.</p><h2>PMC2391110_METHODS_Paragraph7</h2><p class="sentence"><b>Sentence 0:</b> Conventional PCR.</p><p class="sentence"><b>Sentence 1:</b> Conventional PCR was performed with specific primer sets designed for different splice forms as described by and using Taq DNA polymerase (Qiagen).</p><p class="sentence"><b>Sentence 2:</b> The primer pair Muc1-F (GCACTCACCATAGCACG) and Muc1-R (GGCCAGAGTCAATTGTAC) distinguish between MUC1-TM and a secreted MUC1 isoform, containing the VNTR but lacking the transmembrane domain and cytoplasmic tail, MUC1/secreted (SEC).</p><p class="sentence"><b>Sentence 3:</b> PCR products were separated by electrophoresis on 3.5% agarose gel and visualised under UV light with ethidium bromide.</p><h2>PMC2391110_METHODS_Paragraph8</h2><p class="sentence"><b>Sentence 0:</b> Measurement of CA15-3.</p><p class="sentence"><b>Sentence 1:</b> CA15-3 levels were determined using the IMMULITE 2000 BR-MA (CA15-3) assay (Diagnostic Products Corporation, Los Angeles, CA, USA) according to the manufacturer's instructions.</p><p class="sentence"><b>Sentence 2:</b> The assay is a sandwich ELISA utilising the monoclonal antibodies <span class="highlight">115D8</span> and DF3.</p><p class="sentence"><b>Sentence 3:</b> A value above 53 kU l-1 was considered to be outside the normal range of healthy persons.</p><p class="sentence"><b>Sentence 4:</b> All assays were performed on coded samples by technical staff unaware of the patient's diagnosis.</p><h2>PMC2391110_METHODS_Paragraph9</h2><p class="sentence"><b>Sentence 0:</b> Statistics.</p><p class="sentence"><b>Sentence 1:</b> To test for statistically significant differences biomarkers were transformed to the logarithmic scale on which normal theory statistical estimates (mean, s.d.) and tests (t-tests) were applied.</p><h2>PMC2391110_RESULTS_Paragraph1</h2><p class="sentence"><b>Sentence 0:</b> RESULTS EMA staining by Immunohistochemistry.</p><p class="sentence"><b>Sentence 1:</b> The pattern of staining for all antibodies examined was similar, with accentuated staining detectable on the cell membrane (Figure 1).</p><p class="sentence"><b>Sentence 2:</b> The sensitivity of the different antibody clones for mesothelioma effusion samples ranged from 25% for the VU2G7 and VU4H5 clones to 100% for the Mc5 clone.</p><p class="sentence"><b>Sentence 3:</b> The E29 clone reacted with 16 (of 19) specimens (Table 3).</p><p class="sentence"><b>Sentence 4:</b> Most of the antibodies studied were highly specific for malignant cells with no moderate or strongly stained cells being observed in up to 16 specimens of benign reactive effusion studied (Table 3).</p><p class="sentence"><b>Sentence 5:</b> However the Mc5 clone reacted strongly with over half of the nonmalignant samples (57% false-positive rate).</p><p class="sentence"><b>Sentence 6:</b> The MA695 clone had weak or equivocal staining on two thirds of these control samples (Table 3).</p><p class="sentence"><b>Sentence 7:</b> The pattern of reactivity of the Mc5 clone was membrane accentuated in both the malignant and benign specimens.</p><h2>PMC2391110_RESULTS_Paragraph2</h2><p class="sentence"><b>Sentence 0:</b> MUC1 mRNA is overexpressed in mesothelioma cells.</p><p class="sentence"><b>Sentence 1:</b> The relative expression of the archetypal, full-length, transmembrane MUC1 isoform, MUC1-TM, normalised to GAPDH, was significantly greater in the cells from pleural effusion (P<0.0001) and surgically-excised tumour (P=0.001) mesothelioma samples than that in normal mesothelial cells (Figure 2).</p><p class="sentence"><b>Sentence 2:</b> Median expression levels were two to threefold greater in malignant mesothelioma samples than in normal mesothelial cells.</p><p class="sentence"><b>Sentence 3:</b> Expression of MUC1-TM in cells from mesothelioma effusions were significantly greater (P<0.005) than in effusions with benign reactive mesothelial cells.</p><p class="sentence"><b>Sentence 4:</b> This overexpression was not due to gene amplification as there was no amplification of MUC1 by FISH (data not shown).</p><p class="sentence"><b>Sentence 5:</b> There was no significant difference in relative levels of MUC1-TM between normal mesothelial cells and from cells obtained from patients with effusions of nonmalignant origin (Figure 2).</p><h2>PMC2391110_RESULTS_Paragraph3</h2><p class="sentence"><b>Sentence 0:</b> Alternative splice forms of MUC1.</p><p class="sentence"><b>Sentence 1:</b> Conventional PCR using primers distinguishing MUC1-TM from a secreted isoform of MUC1 which lacks the transmembrane region, MUC1-SEC, demonstrated that approximately half of the samples from malignant mesothelioma cells either from solid tumour or from effusions expressed MUC1-SEC in addition to the transmembrane form (Figure 3).</p><p class="sentence"><b>Sentence 2:</b> Normal mesothelial cells, either from pericardial fluid or from benign reactive effusions predominately expressed the full length MUC1-TM isoform.</p><h2>PMC2391110_RESULTS_Paragraph4</h2><p class="sentence"><b>Sentence 0:</b> Serum CA15-3 level.</p><p class="sentence"><b>Sentence 1:</b> The median concentration of CA15-3 in the serum of patients with malignant mesothelioma was 57.4+-2.5 kU l-1 (range, 13-1321 kU l-1); 35% (17 out of 49) of mesothelioma patients had serum CA15-3 above the upper limit of normal (53 kU l-1) (Figure 4A).</p><p class="sentence"><b>Sentence 2:</b> Though patient numbers were limited, mesothelioma patients with a sarcomatoid or mixed histology had lower serum concentrations of CA15-3 than mesothelioma patients with a predominately epitheliod histology (Figure 4B).</p><h2>PMC2391110_RESULTS_Paragraph5</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> Median CA15-3 concentration in patients with benign asbestos-related disease (either asbestosis and/or pleural plaques) was 38+-6 kU l-1 (range, 14-155 kU l-1); 28% (9 out of 32) of patients with asbestos-related benign disease had CA15-3 levels above the upper limit of normal.</p><p class="sentence"><b>Sentence 2:</b> There was no significant difference in serum CA15-3 between patients with mesothelioma and patients with benign asbestos-related lung and pleural disease.</p><p class="sentence"><b>Sentence 3:</b> One of 10 patients with non-malignant pleural effusions examined had elevated serum <span class="highlight">CA15-3.</span></p><p class="sentence"><b>Sentence 4:</b> Median CA15-3 concentration in the serum of healthy controls was 22+-2 kU l-1 (range, 9-43 kU l-1).</p><p class="sentence"><b>Sentence 5:</b> Median concentrations of CA15-3 were significantly higher in the serum of mesothelioma patients compared to patients with benign effusions (P=0.016) and normal controls (P<0.0001) (Figure 4A).</p><h2>PMC2391110_RESULTS_Paragraph6</h2><p class="sentence"><b>Sentence 0:</b> Effusion CA15-3 level.</p><p class="sentence"><b>Sentence 1:</b> Levels of CA15-3 in pleural effusions of patients with malignant mesothelioma ranged from undetectable to 2614 kU l-1, with a median of 36.9+-3.9 kU l-1.</p><p class="sentence"><b>Sentence 2:</b> Approximately 38% (20 out of 52) of mesothelioma patients had CA15-3 levels above the upper limit of normal (53 kU l-1) (Figure 4C).</p><p class="sentence"><b>Sentence 3:</b> Mesothelioma patients with sarcomatoid histology had lower levels of CA15-3 in their effusion than mesothelioma patients with predominately epitheliod histology, although there was no significant difference between the groups (Figure 4D).</p><h2>PMC2391110_RESULTS_Paragraph7</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> The median CA15-3 concentration in the effusions of patients with nonmesothelioma malignancies was 24+-6 kU l-1 (range, 13-191 kU l-1); CA15-3 was elevated in 24% (6 out of 25) of effusions from these patients.</p><p class="sentence"><b>Sentence 2:</b> There was no significant difference in the effusion CA15-3 levels between the groups of patients with mesothelioma or other malignancy.</p><p class="sentence"><b>Sentence 3:</b> Median CA15-3 in effusions of nonmalignant origin was 8.5+-2.9 kU l-1 (range from undetectable to 42 kU l-1).</p><p class="sentence"><b>Sentence 4:</b> CA15-3 levels were statistically significantly higher in the effusions of patients with mesothelioma than those with benign effusions (P=0.004) (Figure 4C).</p><h2>PMC2391110_TITLE_Paragraph1</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> Overexpression and altered glycosylation of MUC1 in malignant mesothelioma.</p><h2>PMC9334053_ABSTRACT</h2><p class="sentence"><b>Sentence 0:</b> RUNX3-Regulated GALNT6 Promotes the Migration and Invasion of Hepatocellular Carcinoma Cells by Mediating O-Glycosylation of MUC1.</p><p class="sentence"><b>Sentence 1:</b> Background Hepatocellular carcinoma (HCC) is a leading cause of cancer-related death worldwide.</p><p class="sentence"><b>Sentence 2:</b> Dysregulation of messenger RNAs (mRNA) has been recognized to be associated with HCC carcinogenesis and development.</p><p class="sentence"><b>Sentence 3:</b> <span class="highlight">Polypeptide</span> <span class="highlight">GalNAc</span> Transferase 6 (GALNT6), an O-type glycosyltransferase, has been confirmed as tumor promoter in different cancers.</p><p class="sentence"><b>Sentence 4:</b> However, the function of GALNT6 in HCC remains to be studied.</p><p class="sentence"><b>Sentence 5:</b> Methods RT-qPCR and western blot experiments were, respectively, performed for evaluating RNA expressions and protein levels.</p><p class="sentence"><b>Sentence 6:</b> Supported by bioinformatics analysis, mechanism assays were conducted for validating the potential relation between different genes.</p><p class="sentence"><b>Sentence 7:</b> Functional assays were implemented to analyze HCC cell migration and invasion after different transfections.</p><p class="sentence"><b>Sentence 8:</b> Results GALNT6 was aberrantly upregulated in HCC cells.</p><p class="sentence"><b>Sentence 9:</b> Knockdown of GALNT6 could repress HCC cell migration and invasion.</p><p class="sentence"><b>Sentence 10:</b> RUNX3 was verified to bind to GALNT6 promoter and activate GALNT6 transcription.</p><p class="sentence"><b>Sentence 11:</b> GALNT6 depletion led to inhibited O-glycosylation and aggravated degradation of MUC1.</p><p class="sentence"><b>Sentence 12:</b> MUC1 overexpression could rescue the impeded HCC cell migration and invasion induced by GALNT6 knockdown.</p><p class="sentence"><b>Sentence 13:</b> Conclusion To sum up, GALNT6 transcriptionally activated by RUNX3 mediated the O-glycosylation of MUC1, thus exerting promoting influence on HCC cell migration and invasion.</p><h2>PMC9334053_DISCUSS_Paragraph1</h2><p class="sentence"><b>Sentence 0:</b> 4.</p><p class="sentence"><b>Sentence 1:</b> Discussion.</p><p class="sentence"><b>Sentence 2:</b> HCC accounts for more than 90% of total hepatic carcinoma cases.</p><p class="sentence"><b>Sentence 3:</b> As one of the most prevailing malignancies around the world, HCC incidence is still increasing.</p><p class="sentence"><b>Sentence 4:</b> Normally, infection of chronic hepatitis B and hepatitis C exists as the major reason for HCC induction, while addiction to alcohol, dietary exposure to aflatoxins or aristolochic acid, and liver disease metastasis are all risk factors of HCC.</p><p class="sentence"><b>Sentence 5:</b> Advancement in molecular pathology has brought about improvement in HCC diagnosis, detailed subtype classification, and prognosis prediction, which also provides help for application of treatment.</p><h2>PMC9334053_DISCUSS_Paragraph2</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> RNAs, either coding RNAs or noncoding RNAs, are recognized as important effectors in the tumor initiation, progression, and metastasis.</p><p class="sentence"><b>Sentence 2:</b> GALNT6, encoding a number of the <span class="highlight">GalNAc-T</span> family of enzymes, has been discovered as tumor promoters in various diseases.</p><p class="sentence"><b>Sentence 3:</b> Consistently, after we confirmed that GALNT6 was upregulated in HCC tissues and cell lines, we also verified GALNT6 could promote HCC cell migration and invasion, implying GALNT6 was an oncogene in HCC.</p><h2>PMC9334053_DISCUSS_Paragraph3</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> To probe into the underlying mechanism contributing to the upregulation of GALNT6 in HCC cells, we conducted bioinformatics analysis and mechanism assays to ravel out the transcription factor of GALNT6.</p><p class="sentence"><b>Sentence 2:</b> As a result, RUNX3 was discovered as the transcription factor promoting the transcription of GALNT6.</p><p class="sentence"><b>Sentence 3:</b> Subsequently, considering the role of GALNT6 as an O-type glycosyltransferase, we aimed to figure out the downstream protein whose O-glycosylation could be mediated by GALNT6.</p><p class="sentence"><b>Sentence 4:</b> With utilization of STRING and UALCAN databases, as well as Co-IP verification, MUC1, an interacting protein of GALNT6 with aberrantly high expression in HCC was screened out.</p><p class="sentence"><b>Sentence 5:</b> MUC1 has been much investigated regarding to cancer development.</p><p class="sentence"><b>Sentence 6:</b> For instance, MUC1 as a ceRNA participant has been discovered to promote ovarian cancer progression.</p><p class="sentence"><b>Sentence 7:</b> The <span class="highlight">O-glycans</span> of MUC1 has also been reported as a crucial effector in tumor development.</p><p class="sentence"><b>Sentence 8:</b> As MUC1 is a transmembrane mucin glycoprotein expressed on the surface of almost all epithelial cells, vaccine based on MUC1 has been regarded as a promising method for cancer treatment.</p><p class="sentence"><b>Sentence 9:</b> In our study, implementation of VVA pull down assay and western blot assay jointly certified MUC1 O-glycosylation mediated by GALNT6 could lessen the degradation of MUC1 protein in HCC cells.</p><p class="sentence"><b>Sentence 10:</b> The rescue assays further corroborated that GALNT6 exacerbated HCC cell migration and invasion via positively regulating MUC1 expression.</p><h2>PMC9334053_DISCUSS_Paragraph4</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> There are still some limitations in this study included.</p><p class="sentence"><b>Sentence 2:</b> Xenograft models were not involved to analyze the influence of RUNX3/GALNT6/MUC1 axis on HCC progression in vivo, and clinical samples were not obtained for certifying the correlation between GALNT6 expression and pathological features.</p><p class="sentence"><b>Sentence 3:</b> Nevertheless, this study could still provide a novel perspective for understanding the molecular regulatory mechanism implicated in HCC.</p><h2>PMC9334053_DISCUSS_Paragraph5</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> Collectively, our study indicated GALNT6 was overexpressed in HCC.</p><p class="sentence"><b>Sentence 2:</b> GALNT6 transcriptionally upregulated by RUNX3 could mediate the O-glycosylation of MUC1, thus positively regulating MUC1 expression to ameliorate HCC cell migration and invasion.</p><h2>PMC9334053_METHODS_Paragraph1</h2><p class="sentence"><b>Sentence 0:</b> 2.</p><p class="sentence"><b>Sentence 1:</b> Material and Methods 2.1.</p><p class="sentence"><b>Sentence 2:</b> Cell Culture.</p><p class="sentence"><b>Sentence 3:</b> Normal liver cell line (THLE-2) and three HCC cell lines (HuH-7, Hep 3B, and Li-7) were all purchased from American Type Culture Collection (ATCC; Manassas, VA, USA).</p><p class="sentence"><b>Sentence 4:</b> HuH-7 and Hep 3B cells were, respectively, cultured in Dulbecco's modified Eagle's media (DMEM; low <span class="highlight">glucose)</span> containing 10% fetal bovine serum (FBS) and Eagle's Minimum Essential Medium (EMEM) containing 10% FBS.</p><p class="sentence"><b>Sentence 5:</b> Li-7 cells were cultivated in RPMI1640 medium supplemented with 10% FBS.</p><p class="sentence"><b>Sentence 6:</b> BEGM medium with additional 70 ng/mL <span class="highlight">phosphoethanolamine,</span> 5 ng/mL epidermal growth factor (EGF), and 10% FBS was used to culture THLE-2 cells.</p><p class="sentence"><b>Sentence 7:</b> Human embryonic kidney 293T cells were provided by China National Institutes for Food and Drug Control and cultivated in DMEM medium containing 10% FBS and antibiotics.</p><p class="sentence"><b>Sentence 8:</b> All cells were cultured in a humidified atmosphere with 5% CO2 at 37 C.</p><h2>PMC9334053_METHODS_Paragraph10</h2><p class="sentence"><b>Sentence 0:</b> 2.10.</p><p class="sentence"><b>Sentence 1:</b> Coimmunoprecipitation (Co-IP).</p><p class="sentence"><b>Sentence 2:</b> The cells were lysed by cold lysis buffer containing complete protease inhibitors and centrifuged at 14,000 x g for 20 min at 4 C.</p><p class="sentence"><b>Sentence 3:</b> The supernatant from cell lysates was incubated with antibody against IgG or GALNT6.</p><p class="sentence"><b>Sentence 4:</b> The immunocomplexes were then associated with Protein A-Sepharose.</p><p class="sentence"><b>Sentence 5:</b> The proteins were then subjected to analysis through western blot.</p><h2>PMC9334053_METHODS_Paragraph11</h2><p class="sentence"><b>Sentence 0:</b> 2.11.</p><p class="sentence"><b>Sentence 1:</b> Statistical Analysis.</p><p class="sentence"><b>Sentence 2:</b> SPSS 22.0 software was used for the statistical analyses in present study.</p><p class="sentence"><b>Sentence 3:</b> Each experiment was performed for at least three times.</p><p class="sentence"><b>Sentence 4:</b> All data were demonstrated as the mean +- standard derivation (SD).</p><p class="sentence"><b>Sentence 5:</b> Student's t-test was applied for the comparison of two groups, while ANOVA was applied to examine the differences of three or more groups.</p><p class="sentence"><b>Sentence 6:</b> Data was considered to indicate statistical significance with two-sided P < 0.05.</p><h2>PMC9334053_METHODS_Paragraph2</h2><p class="sentence"><b>Sentence 0:</b> 2.2.</p><p class="sentence"><b>Sentence 1:</b> Cell Transfection.</p><p class="sentence"><b>Sentence 2:</b> Small interfering RNAs (siRNAs) including si-GALNT6-1/2/3, si-RUNX3-1/2/3, si-EBF1-1/2/3, and si-CTCF-1/2/3 were generated to knockdown corresponding genes in HCC cells.</p><p class="sentence"><b>Sentence 3:</b> The whole sequence of RUNX3 or MUC1 was subcloned into pcDNA3.1 vectors for overexpression of RUNX3 or MUC1 in HCC cells.</p><p class="sentence"><b>Sentence 4:</b> Scramble siRNAs or empty vector pcDNA3.1 was acquired as negative controls.</p><p class="sentence"><b>Sentence 5:</b> Plasmid transfection was completed with application of Lipofectamine 2000 in strict accordance with user protocol.</p><h2>PMC9334053_METHODS_Paragraph3</h2><p class="sentence"><b>Sentence 0:</b> 2.3.</p><p class="sentence"><b>Sentence 1:</b> Reverse Transcription Quantitative Real-Time Polymerase Chain Reaction (RT-qPCR).</p><p class="sentence"><b>Sentence 2:</b> After isolation of total RNA from cells with TRIzol reagent, the extracted RNA was reversely transcribed into cDNAs with Reverse Transcriptase Kit.</p><p class="sentence"><b>Sentence 3:</b> RNA expression analysis was conducted using qRT-PCR kit.</p><p class="sentence"><b>Sentence 4:</b> The expression of the detected gene was measured by the 2-DeltaDeltaCT method. beta-Actin served as endogenous control.</p><h2>PMC9334053_METHODS_Paragraph4</h2><p class="sentence"><b>Sentence 0:</b> 2.4.</p><p class="sentence"><b>Sentence 1:</b> Transwell Migration and Invasion Assay.</p><p class="sentence"><b>Sentence 2:</b> After transfection of indicated plasmids for 24 h, the cells were added into the apical transwell chamber containing serum-free medium with (for invasion) or without (for migration) Matrigel.</p><p class="sentence"><b>Sentence 3:</b> The basal Transwell chamber was added with 10% FBS.</p><p class="sentence"><b>Sentence 4:</b> After 24 h, the cells in the inside of the chambers were wiped off and the successfully migrated or invaded cells on the outside of the chambers were fixed with methanol and dyed with crystal violet.</p><p class="sentence"><b>Sentence 5:</b> Cell number in 5 randomly selected fields was counted under a microscope.</p><h2>PMC9334053_METHODS_Paragraph5</h2><p class="sentence"><b>Sentence 0:</b> 2.5.</p><p class="sentence"><b>Sentence 1:</b> Wound Healing Assay.</p><p class="sentence"><b>Sentence 2:</b> Transfected cells were planted into a 6-well plate and cultured in a DMEM overnight.</p><p class="sentence"><b>Sentence 3:</b> After scratches were made with a plastic tip, cells were continuously cultured in serum-free DMEM.</p><p class="sentence"><b>Sentence 4:</b> Wound closure was viewed at 0 h and 24 h, respectively.</p><h2>PMC9334053_METHODS_Paragraph6</h2><p class="sentence"><b>Sentence 0:</b> 2.6.</p><p class="sentence"><b>Sentence 1:</b> DNA Pull down Assay.</p><p class="sentence"><b>Sentence 2:</b> For DNA pull down, the biotin-labeled GALNT6 promoter and negative control probes were conjugated with beads for coculturing with the protein extracts from HCC cells for 2 h.</p><p class="sentence"><b>Sentence 3:</b> Proteins were purified and subjected to western blot analysis.</p><h2>PMC9334053_METHODS_Paragraph7</h2><p class="sentence"><b>Sentence 0:</b> 2.7.</p><p class="sentence"><b>Sentence 1:</b> Western Blot.</p><p class="sentence"><b>Sentence 2:</b> After the cells were lysed in RIPA buffer, PROTTOT-1KT and Bradford Protein Assay Kit were, respectively, applied to extract total protein and to detect the concentration of the proteins in cell lysates.</p><p class="sentence"><b>Sentence 3:</b> Later, the proteins treated with 10% SDS-PAGE were transferred onto the PVDF membranes, which were blocked with 5% skimmed milk and then cocultivated with specific primary antibodies against RUNX3, beta-actin, MUC4, or MUC1 overnight at 4 C.</p><p class="sentence"><b>Sentence 4:</b> Subsequently, secondary antibodies were added for further cultivation at room temperature.</p><p class="sentence"><b>Sentence 5:</b> Enhanced chemiluminescence (ECL) detection system was applied to visualize immunoreactive bands.</p><h2>PMC9334053_METHODS_Paragraph8</h2><p class="sentence"><b>Sentence 0:</b> 2.8.</p><p class="sentence"><b>Sentence 1:</b> Luciferase Reporter Assay. 293T cells were cultured in a 24-well plate and grown up to 70% coverage.</p><p class="sentence"><b>Sentence 2:</b> The pGL3 Luciferase Reporter Vectors containing GALNT6 promoter with the wild-type or mutant binding site of RUNX3 were constructed.</p><p class="sentence"><b>Sentence 3:</b> The constructed vectors, as well as pcDNA3.1-RUNX3 or negative control (pcDNA3.1), were transfected into 293 T cells, respectively.</p><p class="sentence"><b>Sentence 4:</b> The luciferase assay was performed according to the manufacturer's protocol.</p><h2>PMC9334053_METHODS_Paragraph9</h2><p class="sentence"><b>Sentence 0:</b> 2.9.</p><p class="sentence"><b>Sentence 1:</b> Chromatin Immunoprecipitation (ChIP).</p><p class="sentence"><b>Sentence 2:</b> ChIP RNA-Binding Protein Immunoprecipitation Kit was applied in ChIP assay for the investigation into the putative correlation between RUNX3 and GALNT6 promoter.</p><p class="sentence"><b>Sentence 3:</b> The crosslinked chromatin was first cut into fragments (200-1000 bp) for immunoprecipitation with primary antibody against IgG or RUNX3, together with magnetic beads.</p><p class="sentence"><b>Sentence 4:</b> Subsequent to elution and purification, the precipitated DNAs were analyzed via RT-qPCR.</p><h2>PMC9334053_RESULTS_Paragraph1</h2><p class="sentence"><b>Sentence 0:</b> 3.</p><p class="sentence"><b>Sentence 1:</b> Results 3.1.</p><p class="sentence"><b>Sentence 2:</b> GALNT6 Facilitates Migration and Invasion of HCC Cells.</p><p class="sentence"><b>Sentence 3:</b> Previous studies have unveiled the oncogenic role of GALNT6 in different cancers, including breast cancer and lung adenocarcinoma.</p><p class="sentence"><b>Sentence 4:</b> Hence, we aimed to explore the putative role of GALNT6 in HCC.</p><p class="sentence"><b>Sentence 5:</b> Based on UALCAN (http://ualcan.path.uab.edu/index.html), GALNT6 was discovered to be aberrantly upregulated in HCC tissues (Figure 1(a)).</p><p class="sentence"><b>Sentence 6:</b> RT-qPCR analysis was then done for the measurement of GALNT6 expression in different cell lines, and the results of which showed GALNT6 expression was much higher in HCC cells than in THLE-2 (Figure 1(b)).</p><p class="sentence"><b>Sentence 7:</b> Furthermore, Li-7 and Hep 3B (with the highest GALNT6 expression) were selected for subsequent functional assays.</p><p class="sentence"><b>Sentence 8:</b> The knockdown efficiency of si-GALNT6 was determined in advance (Figure S1A), and si-GALNT6-1/2 was chosen for the higher efficiency.</p><p class="sentence"><b>Sentence 9:</b> It was suggested in RT-qPCR analysis that GALNT6 depletion led to a decrease in the levels of MMP2 and MMP9 (invasion-related genes), suggesting the suppressive impacts of GALNT6 knockdown on HCC cell invasion (Figure 1(c)).</p><p class="sentence"><b>Sentence 10:</b> Wound healing outcomes indicated that the migratory ability of HCC cells could be overtly weakened by GALNT6 silence (Figure 1(d)).</p><p class="sentence"><b>Sentence 11:</b> The inhibitory influences of GALNT6 knockdown on cell migration and invasion in HCC were further confirmed by Transwell experiments as a decline in migrated and invaded cells could be observed as a result of si-GALNT6-1/2 transfection (Figures 1(e) and 1(f)).</p><p class="sentence"><b>Sentence 12:</b> The above findings together prove that GALNT6, upregulated in HCC tissues and cell lines, can contribute to cell migration and invasion in HCC.</p><h2>PMC9334053_RESULTS_Paragraph2</h2><p class="sentence"><b>Sentence 0:</b> 3.2.</p><p class="sentence"><b>Sentence 1:</b> RUNX3 Is the Transcriptional Activator of GALNT6.</p><p class="sentence"><b>Sentence 2:</b> Next, our emphasis was laid on exploration into the molecular mechanism contributing to the aberrantly high expression of GALNT6 in HCC.</p><p class="sentence"><b>Sentence 3:</b> It was learnt from previous research that transcriptional factors might activate the transcription of the target genes via binding to the corresponding promoters.</p><p class="sentence"><b>Sentence 4:</b> Thus, UCSC (http://genome.ucsc.edu/) was applied to predict transcription factors of GALNT6, and three genes (RUNX3, EBF1, and CTCF) were picked out as the candidate transcription factors for their highest cluster score (Figure 2(a)).</p><p class="sentence"><b>Sentence 5:</b> After determining the respective knockdown efficiency of si-RUNX3, si-CTCF, and si-EBF1 (Figure S1B-D), it was manifested in RT-qPCR results that only RUNX3 depletion resulted in an overt reduction in GALNT6 expression (Figure 2(b)).</p><p class="sentence"><b>Sentence 6:</b> The binding site of RUNX3 and GALNT6 was predicted on JASPAR (https://jaspar.genereg.net/), and the predicted sequence with the highest score was GGAACCACAACA (Figure 2(c)).</p><p class="sentence"><b>Sentence 7:</b> In DNA pull down assay, RUNX3 was discovered to be abundantly pulled down by Bio-GALNT6 promoter WT, suggesting the affinity between RUNX3 and GALNT6 (Figure 2(d)).</p><p class="sentence"><b>Sentence 8:</b> Moreover, according to outcomes of luciferase reporter assay, RUNX3 overexpression could obviously elevate the luciferase activity of pGL3-GALNT6 promoter WT, while that of pGL3-GALNT6 promoter MUT was hardly altered under the same condition, which reflected that RUNX3 could activate the transcription of GALNT6 (Figure 2(e)).</p><p class="sentence"><b>Sentence 9:</b> The binding correlation between RUNX3 and GALNT6 promoter was further confirmed by ChIP assay (Figure 2(f)).</p><p class="sentence"><b>Sentence 10:</b> To conclude, RUNX3 can transcriptionally modulate GALNT6 in HCC cells.</p><h2>PMC9334053_RESULTS_Paragraph3</h2><p class="sentence"><b>Sentence 0:</b> 3.3.</p><p class="sentence"><b>Sentence 1:</b> GALNT6 Mediates O-Glycosylation T of MUC1.</p><p class="sentence"><b>Sentence 2:</b> Based on recent studies, the oncogenic role of GALNT6 in breast cancer and lung adenocarcinoma is dependent on O-glycosylation.</p><p class="sentence"><b>Sentence 3:</b> Therefore, we also tended to explore whether GALNT6 modulation mechanism in HCC cells was associated with O-glycosylation of its downstream protein.</p><p class="sentence"><b>Sentence 4:</b> We firstly searched on STRING (https://string-db.org/) to predict the interacting proteins of GALNT6 and applied UALCAN to predict the expression patterns of the top four proteins with highest score (Figure 3(a)).</p><p class="sentence"><b>Sentence 5:</b> The expression of MUC5AC in LIHC could not be obtained from UALCAN.</p><p class="sentence"><b>Sentence 6:</b> Among the other three candidates, MUC1 and MUC4, upregulated in LIHC tissues, were chosen.</p><p class="sentence"><b>Sentence 7:</b> The results of Co-IP assay revealed the stronger binding between MUC1 and GALNT6 than that between MUC4 and GALNT6, so MUC1 was determined as our research target (Figure 3(b)).</p><p class="sentence"><b>Sentence 8:</b> Then, several O-glycosylation sites in MUC1 were observed based on NetOGlyc-4.0 (https://services.healthtech.dtu.dk/service.php?NetOGlyc-4.0) prediction (Figure 3(c)).</p><p class="sentence"><b>Sentence 9:</b> As illustrated in Figure 3(d), GALNT6 depletion resulted in a decline in MUC1 protein level pulled down by vicia villosa agglutinin (VVA), mirroring that GALNT6 might mediate MUC1 O-glycosylation.</p><p class="sentence"><b>Sentence 10:</b> For further confirmation, Li-7 cells were treated with O-glycosylation inhibitor Benzyl-alpha-GalNAc (BAG).</p><p class="sentence"><b>Sentence 11:</b> According to VVA lection pull down and western blot analysis, GALNT6 overexpression could contribute to MUC1 O-glycosylation (Figure 3(e)).</p><p class="sentence"><b>Sentence 12:</b> Additionally, HCC cells transfected with si-NC or si-GALNT6-1 were subjected to cycloheximide (CHX) treatment and the results of western blot analysis verified that GALNT6 depletion could accelerate MUC1 protein degradation (Figure 3(f)).</p><p class="sentence"><b>Sentence 13:</b> Taken together, GALNT6 mediates O-glycosylation of MUC1 in HCC cells.</p><h2>PMC9334053_RESULTS_Paragraph4</h2><p class="sentence"><b>Sentence 0:</b> 3.4.</p><p class="sentence"><b>Sentence 1:</b> GALNT6 Influences HCC Cell Migration and Invasion via Modulation on MUC1.</p><p class="sentence"><b>Sentence 2:</b> The validity of GALNT6/MUC1 regulatory axis in HCC cells was next investigated by performing rescue experiments.</p><p class="sentence"><b>Sentence 3:</b> MUC1 overexpression efficiency was firstly detected via RT-qPCR (Figure S1E).</p><p class="sentence"><b>Sentence 4:</b> Similarly, RT-qPCR analysis was done to examine changes in MMP2 and MMP9.</p><p class="sentence"><b>Sentence 5:</b> Reduction in MMP2 and MMP9 levels, resulting from GALNT6 silence, was discovered to be recovered by MUC1 overexpression (Figure 4(a)).</p><p class="sentence"><b>Sentence 6:</b> Supported by wound healing assay, it was discovered that MUC1 overexpression could reverse the inhibition on cell migration which was attributed to GALNT6 silence (Figure 4(b)).</p><p class="sentence"><b>Sentence 7:</b> The outcomes of Transwell assays further confirmed that the suppressive influences of GALNT6 depletion on cell migration and invasion could be completely countervailed by MUC1 overexpression (Figures 4(c) and 4(d)).</p><p class="sentence"><b>Sentence 8:</b> To sum up, GALNT6/MUC1 axis is a crucial modulator for HCC cell migration and invasion.</p><h2>PMC9334053_TITLE_Paragraph1</h2><p class="sentence"><b>Sentence 0:</b> Paragraph.</p><p class="sentence"><b>Sentence 1:</b> RUNX3-Regulated GALNT6 Promotes the Migration and Invasion of Hepatocellular Carcinoma Cells by Mediating O-Glycosylation of MUC1.</p></body></html>